frontend/javascripts/libs/error_handling.js

```diff
@@ -132,12 +132,20 @@ class ErrorHandling {
         return;
       }
       if (error == null) {
-        // older browsers don't deliver the error parameter
+        // Older browsers (and apparently Safari) don't deliver the error parameter
         error = new Error(message);
       }
       console.error(error);
       this.notify(error);
 
+      if (error.toString() === "Error: Script error.") {
```

Comment:
@daniel-wer Maybe you have a better idea? I couldn't think of another solution, since Safari doesn't expose anything useful in the error message :thinking:

Reply:
Hmm, that's unfortunate :/ Not sure whether you've seen these:
https://github.com/ant-design/ant-design/issues/15075 is the exact issue, but was closed prematurely. However, a new issue was created: https://github.com/ant-design/ant-design/issues/26621 which mentions that this was fixed in some dependent library in the mean time. Doesn't look like it was really fixed, does it? :/

Reply:
Yes, I saw this, but noticed that the subdependencies are already at the newest version if I'm not mistaken (e.g., `rc-align` is at `4.0.9` according to npm and also according to `yarn.lock`). However, I'll link these issues in the follow-up issue in the hope that there will be a future fix in there :)

Reply:
I triggered an update of the antd dependencies by removing/adding it via yarn, but it doesn't look like any dependencies relevant to this error were updated :/ Let's go with this temporary workaround then :)

Reply:
Cool, thank you! Having up to date sub dependencies probably will probably save us from other troubles sooner or later :)